base_layer/core/src/proof_of_work/diff_adj_manager/diff_adj_storage.rs

```diff
@@ -202,6 +224,9 @@ where T: BlockchainBackend
                 }
             }
             self.sync_data = Some((height_of_longest_chain, best_block));
+            while (self.timestamps.len() > 11) {
```

Comment:
`11` should be made a configurable constant.

Reply:
This should not really be configurable, its a consensus number. But I have added this to the consensus constants.

Reply:
Also, making it configurable makes it easier to write tests